dts/msm8916/rules.mk

```
@@ -4,6 +4,7 @@ DTBS += \
 	$(LOCAL_DIR)/apq8016-samsung-r02.dtb \
 	$(LOCAL_DIR)/apq8016-samsung-r07.dtb \
 	$(LOCAL_DIR)/msm8216-samsung-r05.dtb \
+	$(LOCAL_DIR)/msm8916-alcatel-idol347.dtb \
```

`qcom,board-id = <8 1>` is https://github.com/msm8916-mainline/android_kernel_qcom_msm8916/blob/master/arch/arm/boot/dts/qcom/msm8916-mtp-smb1360.dts

```suggestion
	$(LOCAL_DIR)/msm8916-mtp-smb1360.dtb \
```